pypyr-go-terra/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# note ENTRYPOINT is still pypyr, meaning you can run this container as if it's
+# the pypyr binary.
+# be aware that the WORKDIR is NOT the GOPATH.
+# the WORKDIR is /src, which is the pypyr working dir. this will let you mount a
+# volume with a pipelines dir in it without hiding the go path.
+# build me like this from repo root:
+# docker build -t pypyr/pypyr-go-terra -t pypyr/pypyr-go-terra:latest -t pypyr/pypyr-go-terra:0.11 -t pypyr/pypyr-go-terra:0.11.8 -f pypyr-go-terra/Dockerfile .
+
+FROM pypyr/pypyr-go:pypyr1.0.0-go1.11
+
+# get the Terraform executable as zip
+RUN wget https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip
```

Comment:
if you declare `ARG terraformversion=0.11.8` on top after the FROM, then you only need to update it in one place when bumping versions. . .